lib/render/site-award.php

```diff
@@ -66,46 +66,54 @@ function RenderSiteAwards($userAwards): void
 function RenderAwardGroup($awards, $title): void
 {
     $numItems = is_countable($awards) ? count($awards) : 0;
-    if ($numItems == 0) {
+    $numHidden = 0;
+    foreach ($awards as $award) {
+        if ($award['DisplayOrder'] < 0) {
+            $numHidden++;
+        }
+    }
+    if ($numItems == $numHidden) {
+        // No items to show
         return;
     }
 
     $icons = [
         "Game Awards" => "👑🎖️",
         "Event Awards" => "🌱",
-        "Site Awards" => "⬩",
+        "Site Awards" => "🌐",
```

Comment:
I'd like to replace all those emojis with svg icons down the line for visual consistency's sake.